src/storage/src/internal_control.rs

```diff
@@ -7,12 +7,15 @@
 //! workers from individual operators/workers.
 
 use std::collections::BTreeMap;
+use std::sync::Arc;
 use std::time::Instant;
 
 use mz_repr::{GlobalId, Row};
 use mz_storage_client::controller::CollectionMetadata;
 use mz_storage_client::types::sinks::{MetadataFilled, StorageSinkDesc};
 use mz_storage_client::types::sources::IngestionDescription;
+use once_cell::sync::OnceCell;
```

Comment:
Is there a reason why we can't use `OnceCell` from stdlib?

Reply:
due to ignorance 😅, let me take a look

Reply:
After a refactor not using once_cell anymore.